cypress/features/regression/preview.feature

```diff
@@ -3,17 +3,17 @@ Feature: Preview default component
 
   @positive
   Scenario: Enable loading checkbox for a Preview component
-    When I open default "Preview" component in noIFrame with "preview" json from "commonComponents" using "loading" object name
+    When I open basic "Preview Test" component in noIFrame with "preview" json from "commonComponents" using "loading" object name
```

Comment:
this TC (don't forget about `.json` file) could be deleted - it is covered by `chromatic`